synapse/config/server.py

```diff
@@ -319,6 +328,15 @@ def default_config(self, server_name, data_dir_path, **kwargs):
         #
         #use_presence: false
 
+        # whether to require users to authenticate in order to query /profile
+        # endpoints via CS API. this is a workaround in advance of MSC1301 landing
+        #auth_profile_reqs: false
```

Comment:
we're adopting a convention that you put the *opposite* to the default in the sample config, so that using it is just a matter of uncommenting it. IOW: this should be `auth_profile_reqs: true`

Reply:
that seems incredibly unintuitive to me; i would expect commented config to show the defaults.

Reply:
well, it's what we've decided to do. Let's take it elsewhere if you'd like to discuss further.